code/game/objects/items.dm

```diff
@@ -415,6 +419,12 @@ var/global/image/fire_overlay = image("icon" = 'icons/goonstation/effects/fire.d
 //The default action is attack_self().
 //Checks before we get to here are: mob is alive, mob is not restrained, paralyzed, asleep, resting, laying, item is on the mob.
 /obj/item/proc/ui_action_click(mob/user, actiontype)
+	if(actiontype == /datum/action/item_action/remove_tape)

```

Comment:
Erm. 

Is there a better way to implement this on the item_action instead of this? This seems horribly snowflakey.

Reply:
Oh right, haha. There almost certainly will be!